test/specs/annotation/tests/03.table_display.js

```diff
@@ -636,13 +639,21 @@ exports.execute = function (options) {
             it ('should be able to access options in annotation.', function (done) {
                 options.ermRest.resolve(tableCompactOptionsEntityUri, {cid: "test"}).then(function (ref) {
                     var ref = ref.contextualize.compact
-                    expect(ref.display.collapseToc).toBeTruthy("Collapse ToC compact missmatch");
-                    expect(ref.display.hideColumnHeaders).toBeTruthy("Hide Column Headers compact missmatch");
+                    expect(ref.display.collapseToc).toBeTruthy("Collapse ToC compact mismatch");
+                    expect(ref.display.hideColumnHeaders).toBeTruthy("Hide Column Headers compact mismatch");
+                    expect(ref.display.facetPanelOpen).toBeTruthy("Facet Panel Open compact mismatch"); // should be true based on default value
```

Comment:
Shouldn't this be `false`? You're defining the config to only open on association, so it shouldn't be based on the default. right?